docs/core/app4.md

```diff
@@ -14,9 +14,9 @@ The Cosmos-SDK is one such framework.
 While we've already seen `DeliverTx`, the workhorse of any ABCI application,
 here we will introduce the other ABCI requests sent by Tendermint, and
 how we can use them to build more advanced applications. For a more complete
-depiction of the ABCI and how its used, see 
+depiction of the ABCI and how its used, see
 [the
-specification](https://github.com/tendermint/abci/blob/master/specification.md)
+specification](https://github.com/tendermint/tendermint/abci/blob/master/specification.md)
```

Comment:
404 but i'll fix in https://github.com/tendermint/tendermint/issues/1847 because it won't be at ^ this location

link should be REPO/blob/master/abci/spec